person_capture/gui_app.py

```
@@ -10383,7 +10517,30 @@ def _set_ref_paths(self, paths: list[str]) -> None:
+        if not getattr(self, "_updating_refs", False) and hasattr(self, "ref_edit"):
+            raw = self.ref_edit.text().strip()
+            if raw:
+                paths.extend(part.strip() for part in raw.split(";") if part.strip())
+
+        unique: list[str] = []
+        seen: set[str] = set()
+        for path in paths:
+            try:
+                display_path = os.path.normpath(os.path.abspath(path))
+                key = os.path.normcase(display_path)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)